bemani/backend/sdvx/vividwave.py

```diff
@@ -0,0 +1,4135 @@
+                'course_type': 5,
+                'tracks': [
+                    {
+                        'id': 1334,
+                        'type': self.CHART_TYPE_ADVANCED,
+                    },
+                    {
+                        'id': 610,
+                        'type': self.CHART_TYPE_ADVANCED,
+                    },
+                    {
+                        'id': 1033,
+                        'type': self.CHART_TYPE_ADVANCED,
+                    },
+                ],
+            },
+        ]
+
+    def handle_game_sv5_common_request(self, request: Node) -> Node:
+        game = Node.void('game')
+
+        limited = Node.void('music_limited')
+        game.add_child(limited)
+
+        # Song unlock config
+        game_config = self.get_game_config()
+        if game_config.get_bool('force_unlock_songs'):
+            ids = set()
+            songs = self.data.local.music.get_all_songs(self.game, self.version)
+            for song in songs:
+                if song.data.get_int('limited') in (self.GAME_LIMITED_LOCKED, self.GAME_LIMITED_UNLOCKABLE):
+                    ids.add((song.id, song.chart))
+
+            for (songid, chart) in ids:
+                info = Node.void('info')
+                limited.add_child(info)
+                info.add_child(Node.s32('music_id', songid))
+                info.add_child(Node.u8('music_type', chart))
+                info.add_child(Node.u8('limited', self.GAME_LIMITED_UNLOCKED))
+
+        # Catalog, maybe this is for the online store?
+        catalog = Node.void('catalog')
+        game.add_child(catalog)
+
+        for item in []:  # type: ignore
+            info = Node.void('info')
+            catalog.add_child(info)
+            info.add_child(Node.u8('catalog_type', 0))
+            info.add_child(Node.u32('catalog_id', 0))
+            info.add_child(Node.u32('discount_rate', 0))
+
+        # Event config
+        event = Node.void('event')
+        game.add_child(event)
+
+        def enable_event(eid: str) -> None:
+            evt = Node.void('info')
+            event.add_child(evt)
+            evt.add_child(Node.string('event_id', eid))
+
+        if not game_config.get_bool('disable_matching'):
+            # Matching enabled events
+            enable_event("MATCHING_MODE")
+            enable_event("MATCHING_MODE_FREE_IP")
+        enable_event("ICON_FLOOR_INFECTION")
+        enable_event("ICON_POLICY_BREAK")
+        enable_event("ACHIEVEMENT_ENABLE")
+        enable_event("VOLFORCE_ENABLE")
+        enable_event("CONTINUATION")
+        enable_event("TENKAICHI_MODE")
+        enable_event("SERIALCODE_JAPAN")
+        enable_event("DEMOGAME_PLAY")
+        enable_event("TOTAL_MEMORIAL_ENABLE")
+        enable_event("EVENT_IDS_SERIALCODE_TOHO_02")
+        enable_event("KONAMI_50TH_LOGO")
+        enable_event("PREMIUM_TIME_ENABLE")
+        # enable_event("SKILL_ANALYZER_ABLE")
+        # Uncomment when skill analyser courses are imported
```

Comment:
I'm guessing no ;P

Reply:
In my defense, transcribing those festo courses made me not want to do it again lmao

Reply:
Yeah, I never want to do it again either, but I did it for all the games that had courses ;_;